project.mml

```diff
@@ -2343,6 +2349,7 @@ Layer:
           WHERE building IS NOT NULL
             AND building NOT IN ('no')
             AND name IS NOT NULL
+            AND NOT tags->'tower:type' IN ('bell_tower')
```

Comment:
Better `NOT IN`, as above with `building NOT IN ('no')`.

Reply:
Changed